src/deepsparse/pipeline_loggers/manager.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A manager that oversees all the pipeline loggers
+"""
+
+from typing import Any, Dict, List, Set, Union
+
+from deepsparse import PipelineLogger
+from deepsparse.pipeline_loggers.prometheus_pipeline_logger import (
+    IDENTIFIER as PROMETHEUS_IDENTIFIER,
+)
+from deepsparse.pipeline_loggers.prometheus_pipeline_logger import PrometheusLogger
+from deepsparse.timing.timing_schema import InferenceTimingSchema
+
+
+__all__ = ["LoggerManager"]
+
+SUPPORTED_LOGGERS = {PROMETHEUS_IDENTIFIER}
+
+
+class LoggerManager:
```

Comment:
Suggestions:
1. Subclass from PipelineLogger
2. Rename to something like `MultiLogger` or `SequentialLogger` or something that captures that this is a pipeline logger built up of a list of other pipeline loggers

Reply:
Mainly to make it clear to readers & others that that is what it is. I see it the same as if PrometheusLogger didn't subclass PipelineLogger

Reply:
Ok, this makes sense.